package.json

```diff
@@ -82,6 +82,21 @@
                             "program"
                         ],
                         "properties": {
+                            "host": {
+                                "type": "string",
+                                "description": "Host dashmips is running on",
+                                "default": "localhost"
+                            },
+                            "port": {
+                                "type": "number",
+                                "description": "Port dashmips is running on",
+                                "default": 2390
+                            },
+                            "cwd": {
+                                "type": "string",
+                                "description": "Path to terminal",
+                                "default": "${workspaceFolder:interpreter}"
```

Comment:
I think this should just be `${workspaceFolder}` since most users won't have an interpreter folder alias